content/en/blog/2019/alb-ingress-gateway-iks/index.md

```diff
@@ -0,0 +1,394 @@
+---
+title: Securely direct traffic from IBM Cloud Kubernetes Service Ingress to Istio Ingress Gateway
+description: Configure the IBM Cloud Kubernetes Service Application Load Balancer to direct traffic to Istio Ingress gateway with mutual TLS.
+subtitle: Configure the IBM Cloud Kubernetes Service Application Load Balancer to direct traffic to Istio Ingress gateway with mutual TLS
+publishdate: 2019-11-07
+attribution: Vadim Eisenberg (IBM)
+keywords: [traffic-management,ingress,file-mount-credentials,iks]
+---
+
+In this blog post I show how to configure the [Ingress Application Load Balancer (ALB)](https://cloud.ibm.com/docs/containers?topic=containers-ingress-about)
+on [IBM Cloud Kubernetes Service (IKS)](https://www.ibm.com/cloud/kubernetes-service/) to direct traffic to the Istio
+ingress gateway, while securing the traffic between them using {{< gloss  >}}mutual TLS authentication{{< /gloss >}}.
+
+When you use IKS without Istio, you may control your ingress traffic using the provided
+ALB. This ingress-traffic routing is configured using a Kubernetes
+[Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/) resource with
+[ALB-specific annotations](https://cloud.ibm.com/docs/containers?topic=containers-ingress_annotation). IKS provides a
+DNS domain name for each cluster. The domain name is in the following format: `<cluster_name>.<region>.containers.appdomain.cloud`, for example
+`mycluster.us-south.containers.appdomain.cloud`. In addition to registering a DNS domain, IKS provides a TLS certificate
+that matches the domain, and a private key for the certificate. IKS stores the certificates and the private key in a
+[Kubernetes secret](https://kubernetes.io/docs/concepts/configuration/secret/).
+
+When you start using Istio in your IKS cluster, the recommended way is to use the
+[Istio Ingress Gateway](/docs/tasks/traffic-management/ingress/ingress-control/) instead of using the
+[Kubernetes Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/). One of the main reasons to use
+Istio ingress gateway is the fact the ALB provided by IKS will not be able to
+communicate directly with the services inside the mesh when you enable Istio mutual TLS. During your transition to
+having only Istio ingress gateway as your main entry point, you can continue to use the traditional Ingress for
+non-Istio services while using the Istio ingress gateway for services that are part of the mesh.
+
+IKS provides a convenient way for clients to access Istio ingress gateway by letting you
```

Comment:
This paragraph should be removed IMO as it talks about skipping the ALB and can distract the reader.

Reply:
OK, I see why you talk about this. The instructions leverage both DNS entries. Maybe add a couple of sentences describing what the instructions are doing with these two DNS and their secrets?